packages/documentation/docusaurus.config.ts

```diff
@@ -43,9 +43,9 @@ let withBrandTheme = false;
 
 try {
   const path = require.resolve(
-    '@siemens/ix-corporate-theme/dist/css/corporate-theme.css'
+    '@siemens-ix/corporate-theme/dist/css/corporate-theme.css'
   );
-  console.log('Found optionalDependency @siemens/ix-corporate-theme.');
+  console.log('Found optionalDependency @siemens-ix/corporate-theme.');
```

Comment:
Fixing the path here needs additional fixes here:

https://github.com/siemens/ix/blob/f837fcc114276614adbb8c7cb51e4daee5b40d6f/packages/documentation/scripts/build.ts#L386

https://github.com/siemens/ix/blob/f837fcc114276614adbb8c7cb51e4daee5b40d6f/packages/html-test-app/vite.config.ts#L31

https://github.com/siemens/ix/blob/f837fcc114276614adbb8c7cb51e4daee5b40d6f/packages/ionic-test-app/vite.config.ts#L11

Otherwise previews will no showing the theme anymore.